priv/gettext/pt/LC_MESSAGES/terms.po

```diff
@@ -14,194 +14,156 @@ msgstr ""
 #, elixir-format
 #: service.html.eex:24
 msgid "Both registered and unregistered users have rights to protect their property, privacy, and identity unless they have consented or agree to making their identity public. Most information on this topic can be found in our Privacy Policy."
-msgstr ""
-
+msgstr "Os usuários registrados e não registrados têm direitos de proteger sua propriedade, privacidade e identidade, a menos que tenham consentido ou concordado em tornar sua identidade pública. A maioria das informações sobre este tópico pode ser encontrada em nossa Política de Privacidade."
 #, elixir-format
 #: service.html.eex:28
 msgid "By agreeing to these Terms, registered Users can have access to all of Listudy’ features and are free to use them for their own personal, educational, charitable, or developmental purposes. Listudy itself is also free/libre open-source software licensed under the GNU AGPL. This doesn’t prevent Listudy from having the ultimate decision in what kind of use of the Services is appropriate."
-msgstr ""
-
+msgstr "Ao concordar com estes Termos, os usuários registrados podem ter acesso a todos os recursos do Listudy e são livres para usá-los para seus próprios fins pessoais, educacionais, de caridade ou de desenvolvimento. O próprio Listudy também é um software de código aberto gratuito / libre licenciado sob a GNU AGPL. Isso não impede que a Listudy tome a decisão final sobre que tipo de uso dos Serviços é apropriado."
 #, elixir-format
 #: service.html.eex:8
 msgid "By using this website (the “Site”) and services (together with the Site, the “Services”) offered by Listudy.org (together with our subsidiaries, affiliates, representatives, and directors – collectively “Listudy”, “we”, or “us”) you’re agreeing to these legally binding rules (the “Terms”). You’re also agreeing to our Privacy Policy and Cookie Policy, and agreeing to follow any other rules on the Site."
-msgstr ""
-
+msgstr "Ao usar este site (o "Site") e serviços (juntamente com o Site, os "Serviços") oferecidos por Listudy.org (juntamente com nossas subsidiárias, afiliadas, representantes e diretores - coletivamente "Listudy", "nós", ou “nós”) você concorda com essas regras juridicamente vinculativas (os “Termos”). Você também concorda com nossa Política de Privacidade e Política de Cookies, e concorda em seguir quaisquer outras regras do Site."
```

Comment:
Can you please change the current quotation marks " with the ones used in the msgid (“). The current ones break the format of the file.

Reply:
good catch done!